docs/pages/reference/resources.mdx

```diff
@@ -196,3 +203,9 @@ Roles govern access to databases, SSH servers, Kubernetes clusters, web services
 Login rules contain logic to transform SSO user traits during login.
 
 (!docs/pages/includes/login-rule-spec.mdx!)
+
+### Device
+
+Device contains information identifying a trusted device.
+
+(!docs/pages/includes/device-spec.mdx!)
```

Comment:
````suggestion

```yaml
(!docs/pages/includes/device-spec.yaml!)
```
````
Assuming we change this to a YAML file